script/release.sh

```diff
@@ -62,6 +62,10 @@ EOF
 	)
 	mv "$tarball"{,.asc} "$builddir"
 
+	# Go caches pkg-config pretty heavily (including from previous
+	# script/release.sh builds) so clean the cache before building.
+	go clean -cache
+
 	make -C "$root" PKG_CONFIG_PATH="${prefix}/lib/pkgconfig" COMMIT_NO= static
```

Comment:
It seems that adding `EXTRA_FLAGS="-a"` to `make` should solve the issue of pkg-config caching, and is more subtle/focused than cleaning the cache. Source: https://github.com/golang/go/issues/24544#issuecomment-410127446

Reply:
implemented in #2931